ozaria/site/components/cutscene/PageCutscene/index.vue

```diff
@@ -61,28 +44,20 @@ module.exports = Vue.extend({
 
 <template>
   <layout-chrome>
-    <layout-center-content>
-      <div id="cutscene-player"
-      :style="{ width: width+'px', height: height+'px' }" v-if="vimeoId">
-      <!-- Currently this video is a hard coded example, that will be fetched from Cutscene collection -->
-        <base-video
-          :vimeoId="vimeoId"
-          :width="width"
-          :height="height"
+    <base-video
+      :vimeoId="vimeoId"
+      :style="{ width: 'calc(100vw - 106px)', height: 'calc(100vh - 106px)' }"
 
-          v-on:completed="onCompleted"
-        />
-      </div>
-    </layout-center-content>
+      v-on:completed="onCompleted"
+    />
   </layout-chrome>
-
 </template>
 
 <style lang="sass">
 
-#cutscene-player
-  margin-left: auto
-  margin-right: auto
+  #cutscene-player
```

Comment:
Don't think this is used either?  How do we handle horizontal / vertical center?

Reply:
Commented on old version

Reply:
I was answering your question in parallel. I just updated this to expand correctly and thus center the cutscene.